index.html

```diff
@@ -53,5 +53,143 @@ <h5 class="small-intro">I’am a Software Developer</h5>
+        <h3>Multi-Post Stories</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <div class="tags">
+          <div class="tag-item">css</div>
+          <div class="tag-item">html</div>
+          <div class="tag-item">bootstrap</div>
+          <div class="tag-item">Ruby</div>
+        </div>
+        <button type="button">See Project</button>
+      </div>
+
+      <div class="works-grid">
+        <div class="work-item">
+          <div class="work-item-content">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <div class="tags">
+              <div class="tag-item">css</div>
+              <div class="tag-item">html</div>
+              <div class="tag-item">bootstrap</div>
+              <div class="tag-item">Ruby</div>
+            </div>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+        <div class="work-item">
+          <div class="work-item-content">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <div class="tags">
+              <div class="tag-item">css</div>
+              <div class="tag-item">html</div>
+              <div class="tag-item">bootstrap</div>
+              <div class="tag-item">Ruby</div>
+            </div>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+        <div class="work-item">
+          <div class="work-item-content">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <div class="tags">
+              <div class="tag-item">css</div>
+              <div class="tag-item">html</div>
+              <div class="tag-item">bootstrap</div>
+              <div class="tag-item">Ruby</div>
+            </div>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <!-- about me -->
+    <section id="about-me">
```

Comment:
- [ ] Your page has a horizontal scrollbar. Kindly fix the issue. :+1: 

https://user-images.githubusercontent.com/70483835/185478806-2ecc00e6-dcb9-4b64-aab2-592a0cd13e65.mp4